11.6 Runtime Release (#3882)
astronomer/docs: astro/runtime-release-notes.md

:::

## Astro Runtime 11.6.0

> - Release date: June 25, 2024

Reviewer: Just a reminder that we'll need to fix this date whenever we do cut the release.

> - Release date: June 25, 2024

Reviewer 2: ```suggestion
- Release date: June 28, 2024
```

- Airflow version: 2.9.2

### Early access Airflow bug fixes

> - Fixed a bug by adding `[webserver]update_fab_perms` to the deprecated config list, so that you can now use `[fab]update_fab_perms`, the preferred option, instead ([#40317](https://github.com/apache/airflow/pull/40317))

Reviewer: ```suggestion
- Fixed a bug where FAB config options (e.g. `[fab] update_fab_perms`) were not checking for values in the deprecated webserver config section (e.g. `[webserver] update_fab_perms`) ([#40317](https://github.com/apache/airflow/pull/40317))
```
I think this better reflects what this change actually is. The new one still worked, it was the fallback to the old that was broken.

> - Fixed a bug where FAB config options, such as `[fab] update_fab_perms`, were not checking for values in the deprecated webserver config section, for example, `[webserver] update_fab_perms` ([#40317](https://github.com/apache/airflow/pull/40317))

Reviewer 3: ```suggestion
- Fixed a bug where FAB config options, such as `[fab] update_fab_perms`, were not checking for values in the deprecated webserver config section. For example, `[webserver] update_fab_perms` ([#40317](https://github.com/apache/airflow/pull/40317))
```
